modules/light-clients/08-wasm/keeper/msg_server_test.go

```diff
@@ -80,7 +79,7 @@ func (suite *KeeperTestSuite) TestMsgStoreCode() {
 				expectedEvents := sdk.Events{
 					sdk.NewEvent(
 						"store_wasm_code",
```

Comment:
should we use `EventTypeStoreWasmCode` here?

Reply:
I left the string itself because if the string of the event variable ever changes, then we will catch it in the test.